pallets/schemas/src/mock.rs

```diff
@@ -30,16 +30,23 @@ frame_support::construct_runtime!(
 
 parameter_types! {
 	pub const MaxSchemaRegistrations: SchemaId = 64_000;
+	pub const MaxSchemaSizeBytes: u32 = 100;
 }
 
 impl pallet_schemas::Config for Test {
 	type Event = Event;
 	type WeightInfo = ();
 	type MinSchemaSizeBytes = ConstU32<5>;
-	type MaxSchemaSizeBytes = ConstU32<100>;
+	type MaxSchemaSizeBytes = MaxSchemaSizeBytes;
 	type MaxSchemaRegistrations = MaxSchemaRegistrations;
 }
 
+impl Clone for MaxSchemaSizeBytes {
```

Comment:
why do you need this?

Reply:
For the Bounded vector to be cloned() the size type should also be clone able.

Reply:
I have not seen other pallets do this when passing in the bound from the config.

Reply:
removed